packages/flutter/test/material/dropdown_test.dart

```diff
@@ -4449,4 +4449,66 @@ void main() {
     // The dropdown should still be open, i.e., there should be one widget with 'second' text.
     expect(find.text('second'), findsOneWidget);
   });
+
+  testWidgets(
+    'The previous selected item should be highlighted when reopening dropdown on mobile',
```

Comment:
Nit: Should you add a comment about "This is a regression test for https://github.com/flutter/flutter/issues/70294"?